docker-compose/cassandra-3.11/docker-compose-dev-mode.yml

```diff
@@ -40,11 +40,10 @@ services:
     networks:
       - stargate
     ports:
-      - 9080:9080
+      - 8085:8080
     mem_limit: 2G
     environment:
       - STARGATE_BRIDGE_HOST=coordinator
       - STARGATE_BRIDGE_PORT=8091
```

Comment:
nit: you can also just set `QUARKUS_GRPC_CLIENTS_BRIDGE_HOST` here and remove those env vars from docker compose completely..